entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -l
+
+cp /evaluator.js /github/workspace
+npm install -g jest
+jest --json --outputFile=evaluation.json
+node evaluator.js evaluation.json mapping.json result.json
```

Comment:
What do you think about renaming `mapping.json` to `requirements_mapping.json` to make the file data more explicit?